templates/sites-available/drupal-site.j2

```diff
@@ -198,7 +198,7 @@ server {
     {% endif %}
     {% endif %}
 
-    server_name {{item.server_name}};
+    server_name {{item.server_name}}{% if item.default is defined and item.default %} default{% endif %};
```

Comment:
I meant it as test condition, like this
``server_name {{item.server_name}}{% if item.default|default(false) %}default{% endif %};``